requirements.txt

```diff
@@ -11,3 +11,4 @@ SoundFile; sys_platform == 'win32'
 torch>=1.8.0,<=1.10.1
 torchaudio>=0.8.0,<=0.10.1
 tqdm>=4.42.0
+cupy>=10.0.0
```

Comment:
This is not correct. It has to be added to an extra_requirements.txt to any recipe using the LiGRU, but not on the main requirements. We do not want the users to depend on CuPy if they don't want to.